src/Shared/Recruit.Vacancies.Client/Application/Queues/Messages/DeleteStaleQueryStoreDocumentsQueueMessage.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Esfa.Recruit.Vacancies.Client.Application.Queues.Messages
+{
+    public class DeleteStaleQueryStoreDocumentsQueueMessage
+    {
+        public DateTime? CreatedByScheduleDate { get; set; }
```

Comment:
What does CreatedByScheduleDate mean ?

Reply:
It is the timestamp when the schedule was triggered. Not a very good name, but it is consistent with other messages.